### ISSUE TITLE
Unable to Download Books

### ISSUE THREAD

**Issue by BonezIT** [NimaKhajehpour/OpenBooksDownloader#4]
The download button on book pages doesn't seem to work for me. There is no response and no book is downloaded. There is nothing added to the download page either.

**Comment by NimaKhajehpour** [comment]
Hey there @BonezIT, thanks for downloading the app and using it. And thanks for submitting this issue.
May i ask what version of android are you using? It might be because of the way newer android versions handle storage and storage permissions. The app is on its early stage and there might be bugs, this one for instance. But i will do my best to maintain this app regularly.
I will fix the bug as soon as we addresse the cause of the bug which is most probably the android version.
Thanks again for downloading the app and helping it to grow ❤️

**Comment by BonezIT** [comment]
@NimaKhajehpour Sure thing, my android version is 13. I figure that you're correct since it never prompted me for access to storage. Thanks for the work you've done so far. I'm looking forward to using it. 

![Screenshot_20230804_233006_Settings](https://github.com/NimaKhajehpour/OpenBooksDownloader/assets/94773769/9f6b0a4b-570a-4c49-8cf5-d0412df97f91)

**Comment by NimaKhajehpour** [comment]
Yup my guess was right. The app has this problem because wheb i made the app there was no android 12 or 13 so i did not know that was a thing in new androids. I will update the aop and fixing this bug is my first priority. Thanks again for chiosing my app and thanjs for your patience ❤️
